modules/govuk/manifests/apps/content_data_api/db.pp

```diff
@@ -15,4 +15,18 @@
     extensions          => ['plpgsql'],
     enable_in_pgbouncer => false,
   }
+
+  @@monitoring::checks::rds_config { 'content-data-api-postgresql-primary':
+    memory_warning   => 2,
+    memory_critical  => 1,
+    storage_warning  => 100,
+    storage_critical => 50,
+  }
+
+  @@monitoring::checks::rds_config { 'content-data-api-postgresql-standby':
+    memory_warning   => 2,
+    memory_critical  => 1,
+    storage_warning  => 100,
+    storage_critical => 50,
```

Comment:
Given we've over-provisioned the storage, it might be useful to set higher thresholds for storage?